### PR TITLE
setup traits page

### PR DESCRIPTION
**Pull request by tyeung2017**
relates #28 

set up the basic layout of the traits page

### CODE REVIEW

**Review by KatiaAsh — APPROVED** [review]
:sunglasses:

**Review by Wyrd00 — APPROVED** [review]
sweet!

File: traits.js
Comment (Karyum) [inline traits.js:5]: you can use forEach here

File: traits.js
Comment (Karyum) [inline traits.js:11]: and here

**Review by Karyum — CHANGES_REQUESTED** [review]

File: traits.js
Comment (tyeung2017) [reply traits.js:5]: the code itself is alright ...

Any other reason we must change it

Comment (Karyum) [reply traits.js:5]: http://thejsguy.com/2016/07/30/javascript-for-loop-vs-array-foreach.html

File: traits.js
Comment (Karyum) [reply traits.js:11]: http://thejsguy.com/2016/07/30/javascript-for-loop-vs-array-foreach.html

File: traits.js
Comment (Karyum) [reply traits.js:5]: http://thejsguy.com/2016/07/30/javascript-for-loop-vs-array-foreach.html

Comment (tyeung2017) [reply traits.js:5]: this still fails to convince me. I would say this is more of a personal preference, especially in this case when the code is short and sweet and functioning properly.

BTW, did you really check what the getElementsByClassName returns?
It returns a NODELIST, a HTMLcollection which is array like but not array.
It does not have all the array method, like map, reduce;

Nodelist does have forEach, but it is not supported by IE completely.

So in order to do it your way, either I have to 
1, use es6 spread into the array 
2, use .call to bind the result to array, which causes unnecessary overhead
3, use Array from, which needs Babel as well

Is this the way you want us to rewrite three lines of working code into something with much more lines of code?
If yes I want a solid reason for that

BTW, dear QA, this is not the first time you block the PR without a proper reason(remember you treat ternary as ES6 and block us ?).

We dont mind you blocking for good reasons but please make sure you got the facts right as blocking is time consuming and we got a relatively short build time.

Comment (Karyum) [reply traits.js:5]: So what i get out from this is : 
1) You googled a bunch of stuff and found that you should always use forEach instead of a for loop, so you went to your code to replace it, but learned something new that you didn't know before, which is you can't use .forEach or .map etc. on instances of an arrays :tada: bravo.
2) you still don't understand what is a QA, dude if anything i was being nice yesterday by reviewing you pull requests. how can you call it blocking if i had reviewed before  the time i have should review ( which is class time )??
3) even after all googling you still lying to yourself that for loop is better ( which is 100% not ).
4) Quick tip for the future, if you wanna argue with a QA about something that he did wrong, prove him wrong, don't just say that he is wrong. if you sent that nodelist thing before this PR would have been merged ages ago, but you chose the childish way....
Assign me to this PR again so i know you read this
then i can merge it

Comment (tyeung2017) [reply traits.js:5]: 1. The reason why I separate the argument because I think it is two different things, the nodelist one obviously you don't really look into it. (btw arguments is also one of the weird array-like non array structure)

But still want to hear about your rationale about for loop, unfortunately you still cannot convince me. 

2. Not sure about what your second point is?

3. No mate, even for the link you send to me telling I was right. 

4. I want to know about you views for requesting changes on perfectly working fine codes,  and I don't want to shut you up by telling you missing something obvious. (Unfortunately you still cannot convince me).

Comment (Karyum) [reply traits.js:5]: talk to Eoin and Elias as you said you would do i can't be bothered trying to convince cause you won't listen.

Comment (tyeung2017) [reply traits.js:5]: I did already 

And from Elias:

Looping is a low level construct, it's advisable to operate at as high a conceptual level as possible. Using forEach, map, reduce, filter are all more declarative ways of saying what you want to happen. You're ultimately just interested in iterating over list and doing something at each step. You don't care how that iteration is done, really. Using for loops means you have to care about that implementation detail.

which his point is completely different from what you have in your link.

And actually you did not really make a point to convince me, you just said for loop is bad and dont use for loop and did not prove it. 
That sounds like a cult belief of not using for loop.

The link you gave is telling using for loop has performance gain. 

Even for Elias, he does not completely ban the use of for loop, he used the term "advisable" to use others methods.

Comment (Karyum) [reply traits.js:5]: @tyeung2017 I really can't be bothered arguing with you on this, cause again you won't listen.
you didn't even understand what Elias told you, how are you going to understand/listen to me.

**Pull request merged**
